swap_meet/clothing.py

```diff
@@ -0,0 +1,10 @@
+from swap_meet.item import Item
+
+
+class Clothing(Item):
+    def __init__(self, condition=0):
+        self.category = "Clothing"
+        self.condition = condition
```

Comment:
Because `Clothing` is a child class of `Item`, we can create the instance attributes using inheritance and utilizing the `super()` constructor like so:

```suggestion
class Clothing(Item):
    def __init__(self, condition=0):
        super().__init__(self, category = "Clothing", condition)
```

Reply:
Looks like `super().__init__(self, category = "Clothing", condition)` does not need self.
```
class Clothing(Item):
    def __init__(self, condition=0):
        super().__init__("Clothing", condition)
```

Reply:
You're right! That was my typo, thank you for catching that!